.travis.yml

```diff
@@ -1,6 +1,8 @@
 sudo: required
 services:
 - docker
+before_script:
+  - sed -i "s/__RELEASE_VERSION__/$TRAVIS_TAG/g" tomcat/Dockerfile
```

Comment:
Can we use ```TRAVIS_COMMIT``` when ```TRAVIS_TAG``` is not available? 

[Environment Variables - Travis CI](https://docs.travis-ci.com/user/environment-variables/)

Reply:
Yes, that's probably possible.  Note that ```TRAVIS_TAG``` is also used in the ```after_script```  section to insert the release version in the ```bintray.json``` file.  Also, Travis is currently configured to not deploy the build unless the release was tagged.  I can modify this pull request as you suggested, if that's desired.  Just let me know, thanks!

Reply:
My only concern is with CI showing that build failed. It will only pass if a tag is pushed right?
@valvijo What do you think?